go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/lunemec/zapext
+
+go 1.13
+
+require (
+	github.com/certifi/gocertifi v0.0.0-20200211180108-c7c1fbc02894 // indirect
+	github.com/getsentry/raven-go v0.2.0
```

Comment:
Why do we need both?

Reply:
I dunno. Ran `go mod tidy` but I expect it remained in the test or example.